config/crd/bases/keda.sh_scaledobjects.yaml

```diff
@@ -214,6 +214,9 @@ spec:
                         description: |-
                           MetricTargetType specifies the type of metric being targeted, and should be either
                           "Value", "AverageValue", or "Utilization"
+                        enum:
+                        - AverageValue
+                        - Value
```

Comment:
should the enum contain `Utilization` too? if not, the we should consider removing it from the description to match each other

Reply:
Utilization can't be used for external metrics, so I'd not add it, but I'm not sure if we can modify the description above, let me check